### PR TITLE
Set version range for `build_config`

### PR DESCRIPTION
**Pull request by hpoit**
As recommended by Nate Bosch.

### CODE REVIEW

**Comment by hpoit** [comment]
@googlebot I signed it!

File: pubspec.yaml [hunk 1]
```diff
@@ -9,8 +9,7 @@ authors:
 
 description:
   Supports the creation of page objects that can be shared between in-browser tests
-  and WebDriver tests.
-
+@@ -14,19 +14,19 @@ description:
```
Comment (jakemac53) [inline pubspec.yaml]: Looks like a bad merge or something here

File: pubspec.yaml [hunk 2]
```diff
@@ -20,6 +19,7 @@ dependencies:
   analyzer: '>=0.35.0 <0.37.0'
   build: '>0.12.7 <2.0.0'
   built_value: ^6.1.0
+  build_config: '>=0.3.1 <5.0.0'
```
Comment (jakemac53) [inline pubspec.yaml]: Oh, if there was no constraint previously then we don't need to do this change. Probably a new version needs to be published?

Comment (hpoit) [reply pubspec.yaml]: How do I publish it for them, Jake? I need to get this working today!

Comment (hpoit) [reply pubspec.yaml]: When I run TOH 6 my `webdev` succeeds, I get no messages, and my `http://localhost:53322/index.html` keeps `loading...`. I feel like I am really close!

Comment (jakemac53) [reply pubspec.yaml]: You can't publish a package for somebody - they have to add you to owners.

You can try depending on this package by git repo, I think that looks like this:

```
dependencies:
  pageloader:
    git: https://github.com/google/pageloader.git
```

Comment (hpoit) [reply pubspec.yaml]: Just tried that Jake, the behavior didn't change, the webpage is still loading. Any other ideas?

Comment (hpoit) [reply pubspec.yaml]: Jake, the real issue is that I don't know what the problem is. How do I identify it? Which tests should I run?

Comment (hpoit) [reply pubspec.yaml]: @DrMarcII we need your help!

Comment (natebosch) [reply pubspec.yaml]: > if there was no constraint previously then we don't need to do this change. Probably a new version needs to be published?

There really should be a constraint. Removing it was a mistake.

https://github.com/dart-lang/build/tree/9a8ac4bfa883ebc5cf2bc3f76b6bf1393bcb9a46/build_config#publishing-buildyaml-files

**Comment by natebosch** [comment]
@mk13 can you review?

@hpoit - can you fix the bad merge in the `description` section?

**Comment by mk13** [comment]
Let me know if you want me to merge this as soon as Travis is finished.

**Comment by hpoit** [comment]
Hello, I still get the following:
```

**Comment by hpoit** [comment]
@mk13 the build is failing

**Comment by natebosch** [comment]
@hpoit - the package hasn't been published yet. Someone needs to bump the pubspec version and add a changelog entry, then publish.

https://pub.dev/packages/pageloader still shows `3.2.0`

@mk13 - will you be able to publish?

**Comment by hpoit** [comment]
Thanks @natebosch, @mk13, @jakemac53. As soon as it's published I'll try using it!

**Comment by mk13** [comment]
Published 3.2.1 with this change.

**Comment by hpoit** [comment]
Thanks Max. How come the build is still failing?

**Comment by natebosch** [comment]
@hpoit if the error is different from before this isn't the appropriate place to debug.

The discussion leading to this change was a version solve.

In either case your question is too vague to be actionable. Please try gitter or stack overflow and include more details.

**Comment by mk13** [comment]
@hpoit 

The "build failing" icon on pub is just indicative of the chrome driver vs. chrome versioning I mentioned in PR #193 . I've tried resolving it yesterday directly but looks like I'll have to do some additional adjustments in relation to webdriver package to fix that. 3.2.1 should unblock you if your issue is the version range on build_config.

**Pull request merged**